ooiui/templates/science/index_new.html

```diff
@@ -0,0 +1,517 @@
+<!doctype html>
+<html lang="en">
+<head>      
+    <title>OOI Status Page</title>  
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Main Landing Page For status page">
+    <meta name="author" content="RPS ASA">  
+    <meta charset="UTF-8">
+
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <link href="/lib/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="lib/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="lib/bootstrap/dist/css/bootstrap-theme.min.css">
+    <script src="lib/bootstrap/dist/js/bootstrap.min.js"></script>
+
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.5.2/underscore-min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.0.0/backbone-min.js"></script>
+    
+    <!-- Map Stuff -->
+    <link rel="stylesheet" href="/lib/leaflet/dist/leaflet.css"/>
+    <script src="/lib/leaflet/dist/leaflet.js"></script>
+
+    <script src="/lib/d3/d3.min.js"></script>
+    <link rel="stylesheet" href="/lib/Leaflet.awesome-markers/dist/leaflet.awesome-markers.css">
+    <script src="/lib/Leaflet.awesome-markers/dist/leaflet.awesome-markers.js"></script>  
+
+      <!-- Marker Cluster -->
+    <script src="/lib/leaflet.markercluster/dist/leaflet.markercluster.js"></script>
+    <link rel="stylesheet" href="/lib/leaflet.markercluster/dist/MarkerCluster.css" />
+    <link rel="stylesheet" href="/lib/leaflet.markercluster/dist/MarkerCluster.Default.css" />
+
+    
+    <link rel="stylesheet" href="/css/common/scienceLayout.css">
+    <link rel="stylesheet" href="/lib/metis-menu/dist/metisMenu.css">    
+    <script src="/lib/metis-menu/dist/metisMenu.js"></script>
+
+    <script src="/js/views/common/TocColView.js"></script>
+    <script src="/js/models/common/TocColModel.js"></script>
+    <script src="/js/views/common/MapView.js"></script>
+
+
+
+</head>
+
```

Comment:
yeah sure, i think i can use base as well. i just wanted to get the pull request in for the changes we discussed